modules/collectd/manifests/plugin/rabbitmq.pp

```diff
@@ -1,31 +1,23 @@
 # == Class: collectd::plugin::rabbitmq
 #
-# Sets up a collectd plugin to monitor RabbitMQ.
+# Sets up the NYTimes collectd-rabbitmq collectd plugin to monitor RabbitMQ.
```

Comment:
I would suggest to add the url for the project ( https://github.com/NYTimes/collectd-rabbitmq ? )